contracts/treasury/GovernorVotesBondingCheckpoints.sol

```diff
@@ -0,0 +1,214 @@
+        return uint48(roundsManager().currentRound());
+    }
+
+    /**
+     * @dev Machine-readable description of the clock as specified in EIP-6372.
+     */
+    // solhint-disable-next-line func-name-mixedcase
+    function CLOCK_MODE() public view virtual override returns (string memory) {
+        // TODO: Figure out the right value for this
+        return "mode=livepeer_round&from=default";
+    }
+
+    function _getVotes(
+        address _account,
+        uint256 _timepoint,
+        bytes memory
+    ) internal view override returns (uint256) {
+        require(_timepoint <= clock(), ")_getVotes: future lookup");
+
+        return bondingCheckpoints().getStakeAt(_account, _timepoint);
+    }
+
+    function quorum(uint256 _timepoint) public view virtual override returns (uint256) {
+        require(_timepoint <= clock(), ")_getVotes: future lookup");
+
+        return MathUtils.percOf(bondingCheckpoints().getTotalActiveStakeAt(_timepoint), QUORUM);
+    }
+
+    // Vote counting module (GovernorCountingSimple)
+
+    /**
+     * @dev See {IGovernor-COUNTING_MODE}.
+     */
+    // solhint-disable-next-line func-name-mixedcase
+    function COUNTING_MODE() public pure virtual override returns (string memory) {
+        // TODO: Figure out the right value for this
+        return "support=bravo&quorum=for,abstain";
+    }
+
+    /**
+     * @dev See {IGovernor-hasVoted}.
+     */
+    function hasVoted(uint256 proposalId, address account) public view virtual override returns (bool) {
+        return _proposalVotes[proposalId].hasVoted[account];
+    }
+
+    /**
+     * @dev Accessor to the internal vote counts.
+     */
+    function proposalVotes(uint256 proposalId)
+        public
+        view
+        virtual
+        returns (
+            uint256 againstVotes,
+            uint256 forVotes,
+            uint256 abstainVotes
+        )
+    {
+        ProposalVote storage proposalVote = _proposalVotes[proposalId];
+        return (proposalVote.againstVotes, proposalVote.forVotes, proposalVote.abstainVotes);
+    }
+
+    /**
+     * @dev See {Governor-_quorumReached}.
+     */
+    function _quorumReached(uint256 proposalId) internal view virtual override returns (bool) {
+        (uint256 againstVotes, uint256 forVotes, uint256 abstainVotes) = proposalVotes(proposalId);
+
+        // TODO: Should all vote types count for quorum?
+        uint256 totalVotes = againstVotes.add(forVotes).add(abstainVotes);
+
+        return totalVotes >= quorum(proposalSnapshot(proposalId));
+    }
+
+    /**
+     * @dev See {Governor-_voteSucceeded}. In this module, the forVotes must be at least QUOTA of the total votes.
+     */
+    function _voteSucceeded(uint256 proposalId) internal view virtual override returns (bool) {
+        (uint256 againstVotes, uint256 forVotes, uint256 abstainVotes) = proposalVotes(proposalId);
+
+        uint256 totalVotes = againstVotes.add(forVotes).add(abstainVotes);
```

Comment:
Maybe disconsider abstain votes for quota calculation. Perhaps they should only affect the quorum